public/js/monk-widget.js

```diff
@@ -0,0 +1,34 @@
+(function( $ ) {
+
+	/**
+	 * Function for include icon on select menu.
+	 */
+	$.widget( 'custom.iconselectmenu', $.ui.selectmenu, {
+	 	_renderItem: function( ul, item ) {
+	 		var li = $( '<li>' ),
```

Comment:
Are these global selectors correct?
When selecting all `<li>` elements you can affect the current theme interface.
You may need create an identifier using a `monk-*` CSS class.